<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PGK - Zadanie #7: Modele barw - RGB i HSV</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="RGBGroupBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>125</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string notr="true">Model RGB</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QSlider" name="sliderRed">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>16</width>
       <height>128</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderGreen">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>30</y>
       <width>16</width>
       <height>128</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderBlue">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>16</width>
       <height>128</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelRed">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>160</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelGreen">
     <property name="geometry">
      <rect>
       <x>57</x>
       <y>160</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBlue">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>160</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>490</y>
      <width>125</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zako≈Ñcz</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="HSVGroupBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>125</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Model HSV</string>
    </property>
    <widget class="QSlider" name="sliderHue">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>16</width>
       <height>128</height>
      </rect>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderSat">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>30</y>
       <width>16</width>
       <height>128</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderVal">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>16</width>
       <height>128</height>
      </rect>
     </property>
     <property name="maximum">
      <number>511</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelHue">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>160</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSat">
     <property name="geometry">
      <rect>
       <x>57</x>
       <y>160</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelVal">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>160</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ModeGroupBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>125</width>
      <height>75</height>
     </rect>
    </property>
    <property name="title">
     <string>Tryb</string>
    </property>
    <widget class="QRadioButton" name="RGBModeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Model RGB</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="HSVModeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Model HSV</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
